framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.16 under development
 ------------------------
 
+- Bug #16991: Replace `utf8_encode()` by `Php72::utf8_encode()` (GHopperMSK)
```

Comment:
Is this reference correct? Or better `Request::utf8Encode()`?

Reply:
```suggestion
- Bug #16991: Replace `utf8_encode()` by `Request::utf8_encode()` (GHopperMSK)
```

Reply:
This is private method, it should not be mentioned in changelog, since it cannot be used. ```Removing usage of `utf8_encode()` from `Request::resolvePathInfo()` ``` should be enough.